frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/BreakoutColumnList/BreakoutColumnListItem/BreakoutColumnListItem.module.css

```diff
@@ -88,6 +89,7 @@
       }
 
       .BucketTriggerButton {
+        visibility: visible;
```

Comment:
When hovering over the add button, we need to make the binning label visible as well.

<img width="351" alt="Screenshot 2025-02-24 at 4 34 28 PM" src="https://github.com/user-attachments/assets/b86338f7-546d-44f2-a3fe-748a62b2a89b" />